doc/howto_release.md

```diff
@@ -185,17 +245,23 @@ vim wingrass-maintenance-scripts/grass_copy_wwwroot.sh
 vim wingrass-maintenance-scripts/cronjob.sh       # major/minor release only
 ```
 
-# update addon builder
+## Update addon builder
+
 - https://github.com/landam/wingrass-maintenance-scripts/blob/master/grass_addons.sh (add new release related line)
 
-### Close milestone
+## Close milestone
 
-- Close related milestone: https://github.com/OSGeo/grass/milestones
+For a (final) release (not release candidate), close the related milestone at
```

Comment:
Maybe add a hint what to do with issues/PR yet assigned to this milestone (individually bump to next milestone by editing)?

Reply:
You can actually move them all at once :-)

Reply:
In d7523ac40c, I actually added evaluating the state of the milestone to the assumptions as well.